Tools/MonoGame.Effect.Compiler/Effect/CommandLineParser.cs

```diff
@@ -219,7 +223,14 @@ void ShowError(string message, params object[] args)
             }
             Console.Error.WriteLine(message, args);
             Console.Error.WriteLine();
-            Console.Error.WriteLine("Usage: {0} {1}", name, string.Join(" ", _requiredUsageHelp));
+            Console.Error.WriteLine("Usage: {0} {1}", name, string.Join(" ", _requiredUsageArguments));
```

Comment:
The changes here in the command line parser allow us to show individual descriptions for required command line arguments like so:

![image](https://github.com/user-attachments/assets/86c17de1-8b9f-45a3-aaea-f3a93e22bd18)